### PR TITLE
docs(conference): add React advanced to the conference list

### PR DESCRIPTION
**Pull request by orama254**
## Adding a conference
- [x] Suggestion is not a duplicate
- [x] Suggestion is under the proper category
- [x] Link to conference is included 
- [x] Name of conference is included
- [x] Name and link are spelled correctly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ From Angular and React to WebAssembly; from Progressive Web Apps to JAMstack –
 **City and State**: Munich<br>
 **Country**: Germany
 
+[React Advanced](https://reactadvanced.com/)
+
+A conference about modern web development with React, React Native, GraphQL and TypeScript.Get a chance to connect with the global network of field experts and explore react.js.<br>
```
Comment (jdwilkin4) [inline README.md:33]:
```suggestion
A conference about modern web development with React, React Native, GraphQL and TypeScript. Get a chance to connect with the global network of field experts and explore React.js.<br>
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: README.md (hunk 1 above)
Comment (orama254) [reply README.md]: Hello @jdwilkin4 
made changes based on the review.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**